src/meds/__init__.py

```diff
@@ -1,26 +1,36 @@
 from meds._version import __version__  # noqa
 
 from .schema import (
-    data_schema, label_schema, Label, train_split, tuning_split, held_out_split, patient_split_schema,
-    code_metadata_schema, dataset_metadata_schema, CodeMetadata, DatasetMetadata, birth_code, death_code
+    CodeMetadata,
+    DatasetMetadata,
+    Label,
+    birth_code,
+    code_metadata_schema,
+    data_schema,
+    dataset_metadata_schema,
+    death_code,
+    held_out_split,
+    label_schema,
+    subject_split_schema,
+    train_split,
+    tuning_split,
```

Comment:
I think you also want to add the three new constants you defined, right? `subject_id_field`, `time_field`, and `subject_id_dtype`?